slidechain/cmd/export/export.go

```diff
@@ -126,73 +124,21 @@ func main() {
 		log.Fatal(err)
 	}
 
-	// Get latest block height
-	resp, err = http.Get(*slidechaind + "/get?height=0")
-	if err != nil {
-		log.Fatalf("error getting latest block height: %s", err)
-	}
-	defer resp.Body.Close()
-	if resp.StatusCode/100 != 2 {
-		log.Fatalf("bad status code %d getting latest block height", resp.StatusCode)
-	}
-	block := new(bc.Block)
-	bits, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		log.Fatalf("error reading block from response body: %s", err)
-	}
-	err = block.FromBytes(bits)
-	if err != nil {
-		log.Fatalf("error unmarshaling block: %s", err)
-	}
-
-	resp, err = http.Post(*slidechaind+"/submit", "application/octet-stream", bytes.NewReader(txbits))
+	resp, err = http.Post(*slidechaind+"/submit?wait=1", "application/octet-stream", bytes.NewReader(txbits))
```

Comment:
having the `/submit?wait=1` is awesome! maybe we could add a comment here to explain that this endpoint will submit the tx & block until it's included in the txvm chain (or return error)

Reply:
to that effect, might also be good to build an `*http.Request` so that we can pass in a context object with some timeout to avoid blocking forever in an error case where the tx isn't included